tests/sentry/mail/test_adapter.py

```diff
@@ -216,41 +218,6 @@ def test_get_sendable_user_objects(self):
         assert user4 not in self.adapter.get_sendable_user_objects(project)
 
 
-class MailAdapterBuildSubjectPrefixTest(BaseMailAdapterTest, TestCase):
```

Comment:
why just remove these instead of updating them / writing new ones?

Reply:
At first I removed the tests because I had removed the function but I just added everything back.